nixos/modules/services/networking/openvpn.nix

```diff
@@ -1,12 +1,28 @@
 { config, lib, pkgs, ... }:
 
-with lib;
+# with lib;
 
 let
 
   cfg = config.services.openvpn;
 
   inherit (pkgs) openvpn;
```

Comment:
Shouldn't we take this from a package option? We should change that in another PR.

Reply:
Will use another PR for cleaner diff